.idea/codeStyles/Project.xml

```diff
@@ -1,29 +1,29 @@
-<component name="ProjectCodeStyleConfiguration">
```

Comment:
can you please ignore .idea/ files while sending a PR

Reply:
Ok, will do.